plugin/kubernetes/kubernetes.go

```diff
@@ -262,6 +266,10 @@ func (k *Kubernetes) InitKubeCache() (err error) {
 
 	k.opts.initPodCache = k.podMode == podModeVerified
 
+	k.opts.watchChan = &(k.watchChan)
```

Comment:
`&(k.watchChan)` looks out of place are you sure this is correct?

Reply:
It's needed. due to the ordering of how servers and plugins get intialized.

Reply:
done